x/smart-account/post/post.go

```diff
@@ -97,7 +97,6 @@ func (ad AuthenticatorPostDecorator) PostHandle(
 		}
 
 		// We skip replay protection here as it was already checked on authenticate.
-		// TODO: We probably want to avoid calling this function again. Can we keep this in cache? maybe in transient store?
```

Comment:
Is there a reason for removing this? As in is the plan to never do this? If there is still a plan to do this I saw we keep this with a link to the issue

Reply:
+1

Reply:
Oki will keep it and add and issue as well

https://github.com/osmosis-labs/osmosis/issues/8371

See 77f96607986f2d584f106227801ca6dd5629f934